src/System.Security.Cryptography.OpenSsl/ref/System.Security.Cryptography.OpenSsl.cs

```diff
@@ -67,7 +67,7 @@ public sealed partial class RSAOpenSsl : System.Security.Cryptography.RSA
     }
     public sealed partial class SafeEvpPKeyHandle : System.Runtime.InteropServices.SafeHandle
     {
-        public SafeEvpPKeyHandle(System.IntPtr handle, bool ownsHandle) : base (default(System.IntPtr), default(bool)) { }
+        public SafeEvpPKeyHandle(System.IntPtr handle, bool ownsHandle) { }
```

Comment:
Since the changes you're making had no impact on this regen, please revert to keep change history relevant.